Main/src/java/com/alex/zhurylo/entity/Chapters.java

```diff
@@ -0,0 +1,55 @@
+package java.com.alex.zhurylo.entity;
+
+import java.util.List;
+
+public class Chapters {
```

Comment:
This class represents one chapter, so the naming will be better Chapter instead of Chapters